build.gradle

```diff
@@ -2,12 +2,11 @@
 
 buildscript {
     repositories {
-        mavenLocal()
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.1.1'
```

Comment:
One of the links in your commit message mentions Gradle 3.4; is there a particular reason to move to 3.1 rather than 3.4?

Reply:
Good question. This is just the plugin, which should be the latest, while the actual Gradle being used is 4.4.
https://github.com/telenordigital/connect-android-sdk/pull/114/files#diff-fd9d61e37d939c941e7a62077f38d62dR6
https://developer.android.com/studio/releases/gradle-plugin.html#updating-plugin